src/adapter/src/coord/sequencer/inner/create_index.rs

```diff
@@ -267,11 +235,12 @@ impl Coordinator {
             exported_index_id,
             optimizer_config,
         );
-
-        mz_ore::task::spawn_blocking(
+        let span = Span::current();
+        Ok(StageResult::Handle(mz_ore::task::spawn_blocking(
             || "optimize create index",
             move || {
-                let mut pipeline = || -> Result<(
+                span.in_scope(|| {
```

Comment:
Not for this PR, but I know we talked about moving this piece of tracing logic into `spawn_blocking(...)`, do we know if that's possible?

Reply:
Didn't investigate!